### ISSUE TITLE
another option as object not function parameter

### ISSUE THREAD

**Issue by DanieLazarLDAPPS** [karl-run/react-bottom-scroll-listener#38]
Hi,

greate hook :) I would like to use option as object not function parameters.

```
useBottomScrollListener(
   onBottomCallback,
   {
      offset = 0, 
      debounce = 200,   // Optional debounce in milliseconds, defaults to 200ms
      debounceOptions,  // Overwrite the debounceOptions for lodash.debounce, default to { leading: true }
      triggerOnNoScroll,// If container is too short, enables a trigger of the callback if that happens, defaults to false
   }        
)
```

then you can setup interface option a only one option can be overrided. Now I need to fullfill **offset**, **debounce**, **debounceOptions** and than I am able to set **triggerOnNoScroll** which was important for me.

Thanks,
Daniel

**Comment by karl-run** [comment]
I will add this in an update soon. 😄

**Comment by karl-run** [comment]
The signature is now as follows:

```
function useBottomScrollListener<T extends HTMLElement>(
  onBottom: () => void,
  options?: {
    offset?: number;
    debounce?: number;
    debounceOptions?: DebounceOptions;
    triggerOnNoScroll?: boolean;
  },
): RefObject<T> {
```

and is published as v5.0.0. 

Please give it a try!
